src/logger.ts

```diff
@@ -1,6 +1,10 @@
 import * as winston from "winston"
 
 export const logger = winston.createLogger({
+  format: winston.format.combine(
+    winston.format.timestamp({ format: "YYYY-MM-DD hh:mm:ss A" }),
```

Comment:
Could you, please, change it to 24h format?

Reply:
Yes, I did it.